NSS-Day-Cohort-42/trunch-bunch-truncheons-and-flagons
# Leaderboard functionality completed. Styling initiated.

# Steps to Review

1. `git fetch --all`
1. `git checkout mp-game-functionality`
1. make sure your json is serving
1. check that the live data is rendered to the leaderboard and that the code is all legit
